### ISSUE TITLE
Implement new move-rotate behavior

### ISSUE THREAD

**Issue by OrtnerMichael** [magpylib/magpylib#443]
A lengthy discussion #438  revealed:

- **Conclusion 1:** Take `increment` out and do not replace - its not a worthy feature
- **Conclusion 2:** The `absolute` is cool but not critical for now - add later (maybe)
- **Conclusion 3:** The `start='auto'` (fullpath if input is scalar, append/merge if input is vector) is better than the current `start=-1` because
  - It adds some features like whole path movement without additional syntax/kwargs
  - It provides full previous functionality
  - It brings all most important features to the default level (-1 and append)
  - It is easier to understand/describe the behavior than previous -1 default
- **Conclusion 4:** Padding up a path when `start=x` with `x>pathlength` rather than just appending
- **Conclusion 5:** DO NOT auto-pad orientation when `s.position=x` - it creates some strange behavior
- **Conclusion 6:** match position length when orientation is assigned via setters and vice-versa.
examples (letters represent path indices): 
  - case1: setter is shorter:
    - init condition (`position=abc, orientation=def`)
    - setter: `obj.position=ij`
    - result: `obj.position => ij, obj.orientation => ef`
  - case2: setter is longer:
    - init condition (`position=abc, orientation=def`)
    - setter: `obj.position=ijklmn`
    - result: `obj.position => ijklmn, obj.orientation => deffff`
 - **Conclusion 7:** When setting Collection postion/orientation, relative position of the children has to be kept.

**Comment by Alexboiboi** [comment]
Let's write some examples with expected output

```python
import magpylib as magpy
import numpy as np
from magpylib._src.display.plotly.plotly_base_traces import make_BasePrism
from scipy.spatial.transform.rotation import Rotation as R

def make_wheel(Ncubes=6, height=10, diameter=36, path_len=5, name=None):
    cs_lambda = lambda: magpy.magnet.Cuboid(
        (1, 0, 0), dimension=[height] * 3, position=(diameter / 2, 0, 0)
    )
    s0 = cs_lambda().rotate_from_angax(
        np.linspace(0.0, 360.0, Ncubes, endpoint=False), "z", anchor=(0, 0, 0), start=0
    )
    cs_list = []
    for ind in range(Ncubes):
        s = cs_lambda()
        s.position = s0.position[ind]
        s.orientation = s0.orientation[ind]
        cs_list.append(s)
    c = magpy.Collection(cs_list)
    c.rotate_from_angax(90, "x")
    c.rotate_from_angax(
        np.linspace(90, 360, path_len), axis="z", start=0, anchor=(80, 0, 0)
    )
    c.move(np.linspace((0, 0, 0), (0, 0, 200), path_len), start=0)
    c.style.name = name

    trace = make_BasePrism(
        base_vertices=Ncubes, diameter=diameter + height * 2, height=height * 0.5
    )
    trace_plotly = {**trace, "opacity": 0.5, "color": "blue"}
    c.style.model3d.extra = [dict(backend="plotly", trace=trace_plotly)]
    return c

c1 = make_wheel(name="Magnetic Wheel 1")
c1.set_children_styles(path_show=False)
c2 = make_wheel(name="Magnetic Wheel 2")
c2.style.model3d.extra[0].trace['color'] = 'red'
c2.set_children_styles(path_show=True)
c2.move((-200, 0, 0))
magpy.display(c1, c2, path=1, backend="plotly")

![image](https://user-images.githubusercontent.com/29252289/149373464-13284542-3b93-478e-abbb-ac61338ba959.png)

```
<details>
  <summary>`c1.orientation = None`</summary>

![image](https://user-images.githubusercontent.com/29252289/149373858-5ed1fe42-0c77-41c6-9ca8-ba02b7a5f6da.png)  
</details>

<details>
  <summary>`c1.position = np.array([[50, 0, 100]] * 2)`</summary>

![image](https://user-images.githubusercontent.com/29252289/149373530-c2df0346-e2cd-4bfd-b344-baaf100b74d6.png)
  
</details>

<details>
  <summary>`c1.orientation = R.from_rotvec([[90,0,0],[0,90,0]], degrees=True)`</summary>

![image](https://user-images.githubusercontent.com/29252289/149373947-5e51f6bf-09bb-4a46-934e-a64c184fe286.png)
  
</details>

**Comment by Alexboiboi** [comment]
# closed by #458
